appveyor.yml

```diff
@@ -28,10 +28,6 @@ environment:
       PROJECT_DIR: examples\date
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
 
-    - TOOLCHAIN: "msys-cxx17"
-      PROJECT_DIR: examples\date
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
```

Comment:
Can you comment this out with a link to the AppVeyor failure? Thanks.

Reply:
Done. Check again please if that is ok. Appveyor seems to have no retention / purge policy for logs (they do for artifacts, though). So providing this link seems fine. Or did you mean some other kind of link?

Reply:
Thanks! Yup, that's good